pandas/tests/groupby/test_rows.py

```diff
@@ -0,0 +1,302 @@
+""" Test GroupBy.rows positional grouped indexing GH#42864"""
+
+import random
+
+import pytest
+
+import pandas as pd
+import pandas._testing as tm
+
+
+@pytest.fixture()
```

Comment:
you can put this in the groupby conftest to avoid repetition

Reply:
Done